template/.prettierrc.js

```diff
@@ -1,7 +1,44 @@
 module.exports = {
-  arrowParens: 'avoid',
-  bracketSameLine: true,
-  bracketSpacing: false,
   singleQuote: true,
-  trailingComma: 'all',
-};
+  bracketSpacing: true,
+  semi: false,
+  printWidth: 100,
+  arrowParens: 'avoid',
+  trailingComma: 'none',
```

Comment:
Let's force a trailing comma. This simplifies PRs when you add something to hash/array (1 line modified instead of two).

Reply:
Done